minifi/minifi-nar-bundles/minifi-framework-bundle/minifi-framework/minifi-runtime/src/main/java/org/apache/nifi/minifi/MiNiFi.java

```diff
@@ -75,8 +75,7 @@ public MiNiFi(final NiFiProperties properties, ClassLoader rootClassLoader) thro
         }
 
         Thread.setDefaultUncaughtExceptionHandler((t, e) -> {
-            logger.error("An Unknown Error Occurred in Thread {}: {}", t, e.toString());
-            logger.error("", e);
+            logger.error("An Unknown Error Occurred in Thread {}", t, e);
```

Comment:
@dan-s1 I have not tested this but my concern with this change as is would be does it actually change the behavior?  Specifically calling toString on the exception as I recall grabs pretty much the message.  Whereas just passing the exception to the logger will result in the full stack trace.  We don't always want/both.  Please confirm whether this is changing behavior

Reply:
@exceptionfactory had suggested I do this in PRs for [NIFI-13265](https://issues.apache.org/jira/browse/NIFI-13265) and [NIFI-13266](https://issues.apache.org/jira/browse/NIFI-13266). If that is not the desired behavior I certainly can remove this.

Reply:
In this particular case, the behavior is not changed because the stack trace was already logged in a separate line, so this change looks good.